slasher/detection/metrics.go

```diff
@@ -0,0 +1,25 @@
+package detection
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	doubleProposalsDetected = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "double_proposals_detected",
+		Help: "The # of double propose slashable events detected",
+	})
+	doubleVotesDetected = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "double_votes_detected",
+		Help: "The # of double vote slashable events detected",
+	})
+	surroundingVotesDetected = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "surrounding_votes_detected",
+		Help: "The # of surrounding slashable events detected",
+	})
+	surroundedVotesDetected = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "surrounded_votes_detected",
+		Help: "The # of surrounded slashable events detected",
+	})
```

Comment:
Should we add a "rejected by beacon node pool" metric?

Reply:
Will add in the beacon node side